concurrency/concurrency-tck-runner/src/test/resources/arquillian.xml

```diff
@@ -4,16 +4,16 @@
         http://jboss.org/schema/arquillian
         http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
     <defaultProtocol type="Servlet 5.0" />
+
+    <engine>
+        <property name="deploymentExportPath">target/</property>
+    </engine>
     <container qualifier="default" default="true">
         <configuration>
             <property name="jbossHome">${jboss.home}</property>
-            <property name="javaVmArguments">${debugJvmArgs}</property>
```

Comment:
not sure about removing those, which are passed by the pom.xml, but we can re add these later if needed

Reply:
The `javaVmArguments` I kept. The others won't do anything. Those settings aren't in the WildFly container configuration.

Reply:
I was thinking that tests may need those, but found nothing so far and as I said we can re-add if that's the case

Reply:
Ah yeah. In this case what this does is set properties in the [`ManagedContainerConfiguration`](https://github.com/wildfly/wildfly-arquillian/blob/2d48a523cf5aa82738f633cf9e0bac7c326e4445/container-managed/src/main/java/org/jboss/as/arquillian/container/managed/ManagedContainerConfiguration.java). Basically the parameters are just ignored because there is no setter for them.